_includes/posts/maintenance-table.md

```diff
@@ -1,5 +1,6 @@
 | 0.15 | 2017-09-15 | 2018-10-03  | 2019-08-01 |
 | 0.16 | 2018-02-26 | 2019-05-02  | 2020-02-01 |
-| 0.17 | 2018-10-03 | after v0.19 | 2020-08-01 |
+| 0.17 | 2018-10-03 | 2019-11-24  | 2020-08-01 |
 | 0.18 | 2019-05-02 | after v0.20 | TBA        |
-| 0.19 | TBA*       | after v0.21 | TBA        |
+| 0.19 | 2019-11-24 | after v0.21 | TBA        |
+| 0.20 | TBA*       | TBA         | TBA        |
```

Comment:
Should we mention 0.20 here already?

Reply:
Otherwise the * loses meaning here: https://bitcoincore.org/en/lifecycle/

Reply:
Yes, it's fine with me, I just thought it strange to mention a release that isn't even branched off yet.